runtime/src/lib.rs

```diff
@@ -369,7 +347,7 @@ parameter_types! {
 }
 
 impl treasury::Trait for Runtime {
-	type Currency = Balances;
+	type Currency = StakingAssetCurrency<Self>;
```

Comment:
Bump

Reply:
When changed to `SpendingAssetCurrency<Self>`, it throws this error:
```
256 | impl staking::Trait for Runtime {
    |      ^^^^^^^^^^^^^^ expected struct `srml_generic_asset::SpendingAssetIdProvider`, found struct `srml_generic_asset::StakingAssetIdProvider`
    |
    = note: expected type `srml_generic_asset::imbalances::NegativeImbalance<_, srml_generic_asset::SpendingAssetIdProvider<Runtime>>`
               found type `srml_generic_asset::imbalances::NegativeImbalance<_, srml_generic_asset::StakingAssetIdProvider<Runtime>>
```
not sure what this is yet but I think it could be the reason why it was used in the old plug https://github.com/cennznet/plug-blockchain/blob/47f8ec6c7484b72105141fdc1f3d8b372ef83ef2/node/runtime/src/lib.rs#L191-L192

Reply:
Let's leave for a follow up PR in that case